models/user.go

```diff
@@ -164,7 +164,7 @@ func (u *User) update() error {
 }
 
 // CreateUser try and create a new user
-func CreateUser(email, username, password string) error {
+func createUser(email, username, password string) error {
```

Comment:
make this func unexported. Now you can't call this function outside of package `models`